src/favorites.html

```diff
@@ -18,6 +18,8 @@
     <load ="partials/favorites.html" />
   </section>
 
+  <load ="partials/scroll-to-top.html" />
```

Comment:
```suggestion
  <load data-url="partials/scroll-to-top.html" />
```

i did this changes in my pr